instrumentation/logger/lib/opentelemetry/instrumentation/logger/patches/active_support_logger.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Logger
+      module Patches
+        # Patches for the ActiveSupport::Logger class included in Rails
```

Comment:
What's this all about? Well, when this instrumentation is used with Rails, we get double the logs emitted because of how Rails broadcasts the logs. This workaround is similar to [what we did in New Relic's Ruby agent.](https://github.com/newrelic/newrelic-ruby-agent/pull/1019) 

I'm open to feedback and better solutions.

An additional workaround still needs to be developed for the [Rails 7.1 broadcast logger](https://github.com/newrelic/newrelic-ruby-agent/pull/2252).